SOURCEBUILD.md

```diff
@@ -0,0 +1,237 @@
+# Building from Source
+
+The following instructions are for users wishing to build cuGraph from source code.  These instructions are tested on supported distributions of Linux, CUDA, and Python - See [RAPIDS Getting Started](https://rapids.ai/start.html) for list of supported environments.  Other operating systems _might be_ compatible, but are not currently tested. 
+
+The cuGraph package include both a C/C++ CUDA portion and a python portion.  Both libraries need to be installed in order for cuGraph to operate correctly.
+
+### Prerequisites
+
+Compiler requirement:
+
+* `gcc`     version 5.4+
+* `nvcc`    version 10.0+
+* `cmake`   version 3.12
+
+CUDA requirement:
+
+* CUDA 10.0+
+* NVIDIA driver 396.44+
+* Pascal architecture or better
+
+You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
+
+### Build and Install the C/C++ CUDA components
+
+To install cuGraph from source, ensure the dependencies are met and follow the steps below:
+
+1) A cloned version of the repository
+
+  ```bash
+  # Set the localtion to cuGraph in an environment variable CUGRAPH_HOME
+  export CUGRAPH_HOME=$(pwd)/cugraph
+
+  # Download the cuGraph repo - if you have a folked version, use that path here instead
+  git clone https://github.com/rapidsai/cugraph.git $CUGRAPH_HOME
+
+  cd $CUGRAPH_HOME
+  ```
+
+2) Create the conda development environment
+
+```bash
+# create the conda environment (assuming in base `cugraph` directory)
+
+# for CUDA 10
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.0.yml
+
+# for CUDA 10.1
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.1.yml
+
+# for CUDA 10.2
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.2.yml
+
+# activate the environment
+conda activate cugraph_dev
+
+# to deactivate an environment
+conda deactivate
+```
+
+  - The environment can be updated as development includes/changes the dependencies. To do so, run:
+
+
+```bash
+
+# for CUDA 10
+conda env update --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.0.yml
+
+# for CUDA 10.1
+conda env update --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.1.yml
+
+# for CUDA 10.2
+conda env update --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.2.yml
+
+conda activate cugraph_dev
+```
+
+3) Build and install `libcugraph`. CMake depends on the `nvcc` executable being on your path or defined in `$CUDACXX`.
+
+  This project uses cmake for building the C/C++ library. CMake will also automatically build and install nvGraph library (`$CUGRAPH_HOME/cpp/nvgraph`) which may take a few minutes. To configure cmake, run:
```

Comment:
> CMake will also automatically build and install nvGraph library (`$CUGRAPH_HOME/cpp/nvgraph`) which may take a few minutes.

This is outdated